modules/nodes-adding-bare-metal-host-to-cluster-using-yaml.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-yaml_{context}"]
+= Adding a bare metal host to the cluster using YAML in the web console
+
+You can add bare metal hosts to the cluster in the web console using a YAML file that describes the bare metal host.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
```

Comment:
Same comment as above about the mention of provisioning.

Reply:
As above, suggested rewording: "* Install a RHCOS compute machine on bare metal infrastructure for use in the cluster."